toolkit/tools/srpmpacker/srpmpacker.go

```diff
@@ -941,11 +955,10 @@ func hydrateFromRemoteSource(fileHydrationState map[string]bool, newSourceDir st
 			}
 		}
 
-		cancelled := false
-		cancelled, err = retry.RunWithExpBackoff(func() error {
+		cancelled, internalErr := retry.RunWithExpBackoff(func() error {
```

Comment:
While here, should we switch this to use an explicit `err` to avoid naming collisions?
```go
netErr := network.DownloadFile(url, destinationFile, srcConfig.caCerts, srcConfig.tlsCerts)
if netErr != nil {
	logger.Log.Debugf("Failed an attempt to download (%s). Error: %s", url, netErr )
}

return netErr
```

Reply:
I don't see why not.:)